Assets/Mirror/Runtime/ProfilerData.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Diagnostics;
+using UnityEngine.Events;
+
+namespace Mirror
+{
+    /// <summary>
+    /// Provides profiling information from mirror
+    /// A profiler can subscribe to these events and
+    /// present the data in a friendly way to the user
+    /// </summary>
+    public static class ProfilerData
```

Comment:
maybe call it NetworkDiagnostics or similar?
this could be useful for other addons too, e.g. traffic analysis, dos protection, etc.